src/Hooks/FieldStorageConfigEditFormAlter.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Drupal\file_encrypt\Hooks;
+
+use Drupal\Core\Field\FieldTypePluginManagerInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\encrypt\EncryptionProfileManagerInterface;
+use Drupal\field\FieldStorageConfigInterface;
+use Drupal\file\Plugin\Field\FieldType\FileItem;
+
+class FieldStorageConfigEditFormAlter {
+
+  use StringTranslationTrait;
+
+  /** @var \Drupal\Core\Field\FieldTypePluginManagerInterface */
+  protected $fieldTypeManager;
+
+  /** @var \Drupal\encrypt\EncryptionProfileManagerInterface */
+  protected $encryptionProfileManager;
+
+  /**
+   * Creates a new FieldStorageConfigEditFormAlter instance.
+   *
+   * @param \Drupal\Core\Field\FieldTypePluginManagerInterface $field_type_manager
+   *   The field type plugin manager.
+   * @param \Drupal\encrypt\EncryptionProfileManagerInterface $encryption_profile_manager
+   *   The encryption profile manager.
+   */
+  public function __construct(FieldTypePluginManagerInterface $field_type_manager, EncryptionProfileManagerInterface $encryption_profile_manager) {
+    $this->fieldTypeManager = $field_type_manager;
+    $this->encryptionProfileManager = $encryption_profile_manager;
+  }
+
+  /**
+   * Implements hook_form_field_storage_config_edit_form_alter().
+   */
+  public function alterForm(array &$form, FormStateInterface $form_state) {
+    /** @var \Drupal\field\FieldStorageConfigInterface $field_storage_config */
+    $field_storage_config = $form_state->getFormObject()->getEntity();
+    $field_type_definition = $this->fieldTypeManager->getDefinition($field_storage_config->getType());
+    if (is_subclass_of($field_type_definition['class'], FileItem::class)) {
+      $this->doFormAlter($field_storage_config, $form, $form_state);
+    }
+  }
+
+  /**
+   * @param \Drupal\field\FieldStorageConfigInterface $field_storage_config
+   * @param array $form
+   */
+  protected function doFormAlter(FieldStorageConfigInterface $field_storage_config, array &$form) {
```

Comment:
Should we pass the form state? I generally do that even if I don't immediately need it